src/api/json/catalog.json

```diff
@@ -1615,7 +1615,6 @@
     {
       "name": "JSON-API",
       "description": "JSON API document",
-      "fileMatch": ["*.schema.json"],
```

Comment:
This is a common pattern for local schema file names. I suggest you change this to something less generic

Reply:
That's exactly what this PR does. Or do you mean something other than JSON schema?

Reply:
Sorry, I read it wrong